landingzones/views_api.py

```diff
@@ -227,6 +233,58 @@ def perform_create(self, serializer):
             raise APIException('{}{}'.format(ex_msg, ex))
 
 
+class ZoneUpdateAPIView(
+    ZoneModifyMixin, SODARAPIGenericProjectMixin, UpdateAPIView
+):
+    """
+    Update a landing zone description and user message.
+    **URL:** ``/landingzones/api/update/{LandingZone.sodar_uuid}``
+    **Methods:** ``PATCH``, ``PUT``
+    **Parameters:**
+    - ``assay``: Assay UUID (string)
```

Comment:
Since the assay for an existing zone can not be changed, it should not be a parameter.